srcpkgs/vtk/template

```diff
@@ -22,7 +26,7 @@ maintainer="Piraty <piraty1@inbox.ru>"
 license="BSD-3-Clause"
 homepage="https://www.vtk.org"
 distfiles="https://www.vtk.org/files/release/${version:0:3}/VTK-${version}.tar.gz"
-checksum=1b39a5e191c282861e7af4101eaa8585969a2de05f5646c9199a161213a622c7
+checksum=1c5b0a2be71fac96ff4831af69e350f7a0ea3168981f790c000709dcf9121075
 nocross="hdf5 is nocross"
```

Comment:
I think that if a dependency is nocross, the package should just not build if xbps-src can not build that dependency, so this should not be necessary.

Reply:
If I try building locally with `./xbps-src pkg -a 86_64-musl vtk` I get `ERROR: XBPS_CROSS_TRIPLET is not defined!`. Does that mean we should leave it as is?

Reply:
That's a wrong arch, you missed an x.

Reply:
lol oops it's getting late :)

Reply:
Now for realz: `CMake Error at CMake/vtkCrossCompiling.cmake:7 (find_package):   By not providing "FindVTKCompileTools.cmake" in CMAKE_MODULE_PATH this project has asked CMake to find a package configuration file provided by "VTKCompileTools", but CMake did not find one.`

Reply:
I have removed `nocross` on my side and it definitely cannot crossbuild due to hdf5's templated containing `nocross` as well.

Just like @paper42 said, if a dependency is nocross, the package should not build at all.

It is safe to remove the line with `nocross` here.